gulpfile.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
```

Comment:
Small nit here -- at least as a broad Formidable convention (and same for most of our clients), npm scripts have "won".

Gulp, while a great tool, has persistent problems of plugins being behind the "real" upstream projects and adding complexity / abstraction where it's not needed. Here, we just want to execute mocha. I vote we stick with mocha + `package.json:scripts` until we hit a use case we truly can't support.

Side note -- if we need concurrent execution of things, we've got great tools for that that work with `package.json:scripts`.

@jasonwilson -- Thoughts on ^^^ ?